terraform/providers/gce/prod/terraform.tfvars

```diff
@@ -0,0 +1,88 @@
+#--------------------------------------------------------------
+# General
+#--------------------------------------------------------------
+
+# When using the GitHub integration, variables are not updated
+# when checked into the repository, only when you update them
+# via the web interface. When making variable changes, you should
+# still check them into GitHub, but don't forget to update them
+# in the web UI of the appropriate environment as well.
+
+# If you change the atlas_environment name, be sure this name
+# change is reflected when doing `terraform remote config` and
+# `terraform push` commands - changing this WILL affect your
+# terraform.tfstate file, so use caution
+
+name              = "production"
+artifact_type     = "google.image"
+region            = "us-central1-c"
+sub_domain        = "us-central1-c-prod.gce"
+atlas_environment = "gce-us-central1-prod"
+atlas_gce_global  = "gce-global"
+atlas_token       = "REPLACE_IN_ATLAS"
+atlas_username    = "REPLACE_IN_ATLAS"
+site_public_key   = "REPLACE_IN_ATLAS"
+site_private_key  = "REPLACE_IN_ATLAS"
+site_ssl_cert     = "REPLACE_IN_ATLAS"
+site_ssl_key      = "REPLACE_IN_ATLAS"
+vault_ssl_cert    = "REPLACE_IN_ATLAS"
+vault_ssl_key     = "REPLACE_IN_ATLAS"
+vault_token       = "REPLACE_IN_ATLAS" # No need to update until Vault is configured
+
+#--------------------------------------------------------------
+# Network
+#--------------------------------------------------------------
+
+vpc_cidr          = "10.139.0.0/16"
+azs               = "us-east-1a,us-east-1c,us-east-1e" # AZs are region specific
+private_subnets   = "10.139.1.0/24,10.139.2.0/24,10.139.3.0/24" # Creating one private subnet per AZ
+ephemeral_subnets = "10.139.11.0/24,10.139.12.0/24,10.139.13.0/24" # Creating one ephemeral subnet per AZ
+public_subnets    = "10.139.101.0/24,10.139.102.0/24,10.139.103.0/24" # Creating one public subnet per AZ
+
+# Bastion
+bastion_instance_type = "n1-standard-1"
+
+# NAT
+nat_instance_type = "g1-small"
+
+# OpenVPN - https://docs.openvpn.net/how-to-tutorialsguides/virtual-platforms/amazon-ec2-appliance-ami-quick-start-guide/
+openvpn_instance_type = "n1-standard-1"
+openvpn_ami           = "ami-5fe36434"
```

Comment:
Not sure if there is a GCE equivalent of the OpenVPN AMI, if not, let's just remove all OpenVPN stuff.